test/Microsoft.Azure.Devices.Edge.Test/Module.cs

```diff
@@ -56,33 +60,72 @@ public async Task TempSensor()
                 token);
         }
 
+        [Category("TempSensor")]
         [Test]
         public async Task TempFilter()
         {
-            string sensorImage = Context.Current.TempSensorImage.GetOrElse(DefaultSensorImage);
-            string filterImage = Context.Current.TempFilterImage.Expect(() => new ArgumentException());
+            string filterImage = Context.Current.TempFilterImage.Expect(() => new ArgumentException("tempFilterImage parameter is required for TempFilter test"));
+
+            const string filterModName = "tempFilter";
```

Comment:
"Mod" is not a common abbreviation for module so I wouldn't recommend using it in variable names.

Reply:
` filterModName ` has leveled up to be ` filterModuleName `